Biotech/Patches/ThingDefs_Races/Races_Mechanoids_Light.xml

```diff
@@ -335,8 +335,8 @@
 		<value>
 			<ArmorRating_Blunt>4</ArmorRating_Blunt>
 			<ArmorRating_Sharp>2.5</ArmorRating_Sharp>
-			<AimingAccuracy>1.0</AimingAccuracy>
```

Comment:
Why increase their aiming accuracy at the same time as decreasing their shooting accuracy?

Reply:
They are similar to militors in terms of tier but spawned by a more advanced parent mech, hence the second tier of accuracy. Will bring them down to T1.